### ISSUE TITLE
Bug with using it in the newest version(s) of SvelteKit

### ISSUE THREAD

**Issue by jochemvogel** [TahaSh/svelte-paginate#29]
Probably because of this change: https://github.com/sveltejs/kit/pull/5332 the following error will occur:

```
/node_modules/svelte-paginate/src/index.js:1
export { default as paginate } from './paginate.js'
^^^^^^

SyntaxError: Unexpected token 'export'
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1031:15)
    at Module._compile (node:internal/modules/cjs/loader:1065:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:190:29)
    at ModuleJob.run (node:internal/modules/esm/module_job:185:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
```

Found a related issue: https://github.com/vitejs/vite/issues/7130

Reinstalling did work on the first try, but after that still the same error.

Updating the package to ES Modules obviously doesn't fix the problem and will produce a `Unknown file extension ".svelte"` type error.

`const { paginate, LightPaginationNav } = 'svelte-paginate';` wont' fix it neither.

```
svelte-paginate doesn't appear to be written in CJS, but also doesn't appear to be a valid ES module (i.e. it doesn't have "type": "module" or an .mjs extension for the entry point). Please contact the package author to fix.
```

**Comment by jochemvogel** [comment]
I'm not sure how to fix this (yet), but I think it's good that you're aware that with the newest version(s) of SvelteKit the package isn't working anymore.

**Comment by bas-baskara** [comment]
Hi @jochemvogel,

I think you can solve the issue with this https://github.com/TahaSh/svelte-paginate/issues/23#issuecomment-1191348970

**Comment by jochemvogel** [comment]
> Updating the package to ES Modules obviously doesn't fix the problem and will produce a `Unknown file extension ".svelte"` type error.

Thanks for the suggestion @bas-baskara. But as I said: "Updating the package to ES Modules obviously doesn't fix the problem and will produce a Unknown file extension ".svelte" type error."

**Comment by TahaSh** [comment]
It's fixed in `v0.1.0`.
